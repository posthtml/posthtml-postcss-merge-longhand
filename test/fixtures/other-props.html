<div style="display: flex; padding-top: 1px; padding-right: 2px; padding-bottom: 3px; padding-left: 4px; color: red">test</div>
<div style="padding-top: 1px; padding-right: 2px; padding-bottom: 3px; padding-left: 4px; color: red; background-image: url('{{ $foo }}');">expressions</div>
<div style="border-color: red; background-image: url('{{ $foo }}'); border-width: 1px; border-style: dashed;">expressions mixed</div>
