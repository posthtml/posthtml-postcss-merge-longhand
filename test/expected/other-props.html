<div style="display: flex; padding: 1px 2px 3px 4px; color: red">test</div>
<div style="padding: 1px 2px 3px 4px; color: red; background-image: url('{{ $foo }}');">expressions</div>
<div style="background-image: url('{{ $foo }}'); border: 1px dashed red;">expressions mixed</div>
